ios/Sources/GutenbergKit/Sources/Cache/EditorAssetsLibrary.swift

```diff
@@ -22,10 +22,14 @@ public actor EditorAssetsLibrary {
     /// reused on future calls.
     func loadManifestContent() async throws -> Data {
         let endpoint: URL
+        // The GutenbergKit bundle includes the required `@wordpress` modules
+        let excludeParam = URLQueryItem(name: "exclude", value: "core,gutenberg")
```

Comment:
This leverages the query parameter introduced in https://github.com/Automattic/jetpack/pull/45715. We exclude "core" and Gutenberg so that only third-party plugin assets are returned.